assets/src/higher-order-components/filter-bar/entity-list-filter-bar.js

```diff
@@ -93,50 +102,88 @@ class EntityListFilterBar extends Component {
 	 * @return {Object} rendered grid view icon button
 	 */
 	gridView = ( view, setGridView ) => (
+		<Fragment>
+			<label
+				className="screen-reader-text"
+				htmlFor={ `ee-grid-view-btn-${ this.props.instanceId }` }>
+				{ __( 'list view', 'event_espresso' ) }
+			</label>
+			<IconButton
+				id={ `ee-grid-view-btn-${ this.props.instanceId }` }
+				className={ view === 'grid' ? 'active-list' : '' }
+				icon="screenoptions"
+				tooltip={ __( 'grid view', 'event_espresso' ) }
+				onClick={ setGridView }
+			/>
+		</Fragment>
+	);
+
+	/**
+	 * @param {string} view
+	 * @param {string} perPage
+	 * @param {Function} setDefaultView
+	 * @param {Function} setDefaultPerPage
+	 * @return {Object} rendered grid view icon button
+	 */
+	saveSettings = ( view, perPage, setDefaultView, setDefaultPerPage ) => (
 		<IconButton
-			className={ view === 'grid' ? 'active-list' : '' }
-			icon="screenoptions"
-			tooltip={ __( 'grid view', 'event_espresso' ) }
-			onClick={ setGridView }
+			className={ 'ee-save-filter-settings' }
+			icon={ 'admin-generic' }
+			tooltip={ __(
+				'click to save current "per page" and "view" settings',
+				'event_espresso'
+			) }
+			onClick={ () => {
+				setDefaultView( view );
+				setDefaultPerPage( perPage );
```

Comment:
Is the handling for setDefaultView and setDefaultPerPage still supposed to be here?